sublime-package.json

```diff
@@ -2,297 +2,323 @@
 	"contributions": {
 		"settings": [
 			{
-				"file_patterns": ["/LSP-lemminx.sublime-settings"],
+				"file_patterns": [
+					"/LSP-lemminx.sublime-settings"
+				],
 				"schema": {
-					"properties": {
-						"languages": {
-							"type": "array",
-							"items": [
-								{
+					"$id": "sublime://settings/LSP-lemminx",
+					"definitions": {
+						"PluginConfig": {
+							"properties": {
+								"settings": {
 									"type": "object",
 									"properties": {
-										"languageId": {
+										"xml.trace.server": {
+											"type": "string",
+											"enum": [
+												"off",
+												"messages",
+												"verbose"
+											],
+											"default": "off",
+											"description": "Traces the communication between VS Code and the XML language server.",
+											"scope": "window"
```

Comment:
As you already noted `scope` is not required in ST. Maybe use the chance to remove it?